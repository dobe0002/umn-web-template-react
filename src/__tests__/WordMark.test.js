/* eslint-disable react/jsx-filename-extension */
import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import WordMark from '../HeaderFooter/icons/WordMark';

configure({ adapter: new Adapter() });

describe('WordMark', () => {
  it('Wordmark renders correctly without props', () => {
    const wordMark = shallow(<WordMark />);
    expect(wordMark).toMatchSnapshot();
  });
});
