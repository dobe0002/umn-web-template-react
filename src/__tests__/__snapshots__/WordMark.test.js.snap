// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WordMark Wordmark renders correctly without props 1`] = `ShallowWrapper {}`;
